modules/ebi-metagenomics/hmmer/hmmsearch/meta.yml

```diff
@@ -0,0 +1,49 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "hmmer_hmmsearch"
+description: search for sequences against a HMM profile database
+keywords:
+  - hmm
+  - hmmer
+  - sequence search
+  - hmmsearch
+tools:
+  - "hmmer":
+      description: "Biosequence analysis using profile hidden Markov models"
+      homepage: http://hmmer.org/
+      documentation: http://hmmer.org/documentation.html
+      tool_dev_url: https://github.com/EddyRivasLab/hmmer
+      doi: "10.1371/journal.pcbi.1002195"
+      licence: ["BSD"]
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+  - fasta:
+      type: file
+      description: FASTA file containing amino acid sequences to be searched for in profile
```

Comment:
"Pool of sequences from where hits will be search for with the HMM profile"
or something like that